cmd/cli/docker/docker_run_cli_test.go

```diff
@@ -177,15 +177,6 @@ func TestJobFlagParsing(t *testing.T) {
 			},
 			expectedError: false,
 		},
-		{
-			name:  "with concurrency",
-			flags: []string{"--concurrency=100", "image:tag"},
-			assertJob: func(t *testing.T, j *models.Job) {
-				assert.EqualValues(t, 100, j.Count)
-				defaultTaskAssertions(t, j.Task())
-			},
-			expectedError: false,
-		},
```

Comment:
why are we coupling `model` removal with supporting deprecated flags? Can we please do them separately, and only remove support of deprecated flags by 1.5.0?

Reply:
I can revert this change and land it later, tracking via https://github.com/bacalhau-project/bacalhau/issues/4182